data/mods/ssb/moves.ts

```diff
@@ -157,6 +155,34 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		type: "Normal",
 	},
 
+	// Mad Monty
+	stormshelter: {
+		accuracy: true,
+		basePower: 0,
+		category: "Status",
+		name: "Storm Shelter",
+		shortDesc: "User protects and sets up a substitute.",
+		pp: 5,
+		priority: 4,
+		flags: {},
+		stallingMove: true,
+		volatileStatus: 'protect',
+		onPrepareHit(pokemon) {
+			this.attrLastMove('[anim] Protect');
+			return !!this.queue.willAct() && this.runEvent('StallMove', pokemon);
+		},
+		onHit(pokemon) {
+			pokemon.addVolatile('stall');
+			this.actions.useMove('Substitute', pokemon);
+			this.boost({atk: 1, def: 1, spa: 1, spd: 1, spe: 1, accuracy: 1, evasion: 1}, pokemon);
+			this.add(`c:|${getName('Mad Monty')}|Ope! Wrong button, sorry.`);
+			this.boost({atk: -1, def: -1, spa: -1, spd: -1, spe: -1, accuracy: -1, evasion: -1}, pokemon);
```

Comment:
doubt it matters much but if it happens to be +6 in any stat it will wind up at +5 after this

Reply:
I can add a check to only display the message if none of the user’s stats are maxed positively if needed